cases/index.html

```diff
@@ -106,7 +118,11 @@ <h2>Some of the things we have worked on</h2>
     {% capture crm_content %}
         {% assign crm_cases = cases | where_exp: "item", "item.categories contains 'crm'" %}
           {% for case in crm_cases %}
-            {% include case_element.html url=case.url title=case.title customer=case.customer logo=case.logo categories=case.categories %}
+            {% if forloop.index < 3  %}
+                {% include case_element.html url=case.url title=case.title customer=case.customer logo=case.logo categories=case.categories %}
+            {% else %}
+                {% include case_element.html url=case.url title=case.title customer=case.customer categories=case.categories %}

```

Comment:
I don't think passing the logo url is the right way to differentiate the versions. Maybe add a flag "small" or render a different elelemnt "case_element_small" to create proper variants.

For the small variant, we should also decrease text size and make it overall a bit denser.

For the large variant, we should add 1-3 lines of brief description of the case.